Core/wbDefinitionsSF1.pas

```diff
@@ -18030,18 +18049,18 @@ procedure DefineSF1u;
     wbFormIDCk(BNAM,'Workbench Keyword', [KYWD]),
     wbStructs(FVPA, 'Resources', 'Resource', [
       wbFormIDCk('Resource', [IRES, ALCH]),
-      wbByteArray('Unknown', 4),
-      wbFormID('Unknown')
+      wbInteger('Required Count', itU32),
+      wbUnknown(4)
     ]),
     wbStructs(RQPK, 'Perks', 'Perk', [
       wbFormIDCk('Perk', [PERK]),
-      wbByteArray('Unknown', 4),
-      wbFormID('Unknown')
+      wbInteger('Rank', itU32),
+      wbUnknown(4)
     ]),
-    wbFormID(CNAM),
-    wbUnknown(NNAM),
-    wbUnknown(SNAM),
-    wbUnknown(TNAM),
+    wbFormID(CNAM, 'Icon Source'),
+    wbUnknown(NNAM), // always 2 byte $0000
```

Comment:
should be defined with a fixed length then e.g. wbUnknown(NNAM, 2) and if always present, marked as required